LBHackney-IT/lbh-fss-portal-frontend
# Feature/add gtm

**Description**

Add GTM 

**Test**

- navigate to hackney account within Nudge GTM account https://tagmanager.google.com/?authuser=1#/container/accounts/6002202759/containers/32920927/workspaces/3 and click preview 
- pull this branch 
- run application and make sure **GA - Pageview - All Pages** tag triggers on all page changes

**Ticket**

https://app.clickup.com/t/70t70p
